core/server/data/migration/fixtures/006/01-transform-dates-into-utc.js

```diff
@@ -0,0 +1,201 @@
+                    }
+
+                    if (post.get('updated_at')) {
+                        post.set('updated_at', _private.addOffset(post.get('updated_at')));
+                    }
+
+                    post.set('created_at', _private.addOffset(post.get('created_at')));
+
+                    options.id = post.get('id');
+                    return models.Post.edit(post.toJSON(), options);
+                }).then(function () {
+                    logger.info(messagePrefix + 'Updated datetime fields for Posts!');
+                });
+            });
+        },
+        function () {
+            return models.Users.forge().fetch().then(function (result) {
+                if (result.models.length === 0) {
+                    logger.warn(messagePrefix + 'No Users found');
+                    return;
+                }
+
+                return Promise.mapSeries(result.models, function mapper(user) {
+                    if (user.get('last_login')) {
+                        user.set('last_login', _private.addOffset(user.get('last_login')));
+                    }
+
+                    if (user.get('updated_at')) {
+                        user.set('updated_at', _private.addOffset(user.get('updated_at')));
+                    }
+
+                    user.set('created_at', _private.addOffset(user.get('created_at')));
+
+                    options.id = user.get('id');
+                    return models.User.edit(user.toJSON(), options);
+                }).then(function () {
+                    logger.info(messagePrefix + 'Updated datetime fields for Users!');
+                });
+            });
+        },
+        function () {
+            return models.Subscribers.forge().fetch().then(function (result) {
+                if (result.models.length === 0) {
+                    logger.warn(messagePrefix + 'No Subscribers found');
+                    return;
+                }
+
+                return Promise.mapSeries(result.models, function mapper(subscriber) {
+                    if (subscriber.get('unsubscribed_at')) {
+                        subscriber.set('unsubscribed_at', _private.addOffset(subscriber.get('unsubscribed_at')));
+                    }
+
+                    if (subscriber.get('updated_at')) {
+                        subscriber.set('updated_at', _private.addOffset(subscriber.get('updated_at')));
+                    }
+
+                    subscriber.set('created_at', _private.addOffset(subscriber.get('created_at')));
+
+                    options.id = subscriber.get('id');
+                    return models.Subscriber.edit(subscriber.toJSON(), options);
+                }).then(function () {
+                    logger.info(messagePrefix + 'Updated datetime fields for Subscribers!');
+                });
+            });
+        },
+        function () {
+            return models.Settings.forge().fetchAll().then(function (result) {
```

Comment:
As far as I can tell, this settings code is the same as the block below, and so it can probably be merged with the list of objects which only have created_at & updated_at?